app/templates/direct-award/which-service-won-contract.html

```diff
@@ -0,0 +1,82 @@
+{% extends "toolkit/layouts/_base_page.html" %}
+
+{% block page_title %}Which service won the contract? - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+        "link": url_for('main.index'),
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for('external.buyer_dashboard'),
+        "label": "Your account"
+      },
+      {
+        "link": url_for('direct_award.saved_search_overview', framework_framework=framework.framework),
+        "label": "Your saved searches"
+      },
+      {
+        "link": url_for('direct_award.view_project', framework_framework=framework.framework, project_id=project.id),
+        "label": project.name
+      },
+      {
+        "link": url_for('direct_award.did_you_award_contract', framework_framework=framework.framework, project_id=project.id),
+        "label": "Did you award a contract"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="which-service-won-contract-page">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <header class="page-heading">
+        <h1>
+          Which service won the contract?
+        </h1>
+      </header>      
+
+
+      {% if form.which_service_won_the_contract.options %}
+
+      <form method="POST" action="{{ url_for('direct_award.which_service_won_contract', framework_framework=framework.framework, project_id=project.id) }}">
+        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+        {%
+          with
+          name = "which_service_won_the_contract",
+          type = "radio",
+          value = form.which_service_won_the_contract.value,
+          error = form.which_service_won_the_contract.errors[0],
+          options = form.which_service_won_the_contract.options
+        %}
+          {% include "toolkit/forms/selection-buttons.html" %}
+        {% endwith %}
+
+        {%
+          with
+          type = "save",
+          label = "Save and continue"
+        %}
+          {% include "toolkit/button.html" %}
+        {% endwith %}
+      </form>
+
+      {% else %}
+        <div class="dmspeak">
+          <p>You cannot award this contract as there were no services matching your requirements.</p>
```

Comment:
If there are no services, should we allow the user to select 'Yes' to 'Did you award a contract?'?

Reply:
You can still access the page without completing the first bit